glide.yaml

```diff
@@ -1,25 +1,24 @@
 package: github.com/openshift/cluster-machine-approver
 import:
 - package: k8s.io/api
-  version: kubernetes-1.11.1
+  version: kubernetes-1.14.0
   subpackages:
   - certificates/v1beta1
   - core/v1
 - package: k8s.io/apimachinery
-  version: kubernetes-1.11.1
+  version: kubernetes-1.14.0
   subpackages:
   - pkg/fields
   - pkg/util/runtime
   - pkg/util/wait
 - package: k8s.io/client-go
-  version: kubernetes-1.11.1
+  version: kubernetes-1.14.0
   subpackages:
   - kubernetes
   - tools/cache
   - tools/clientcmd
   - util/workqueue
-- package: github.com/golang/glog
 - package: github.com/openshift/cluster-api
-  version: release-4.0
+  version: openshift-4.2-cluster-api-0.0.0-alpha.4
 - package: sigs.k8s.io/controller-runtime
```

Comment:
Still need to drop.

Reply:
_oh_ last comment I _thought_  you meant me to drop the cluster-api above, then was confused bc we're calling that, so then thought you meant to drop my _change_ to the version.  Now I see you meant to drop the controller-runtime, bc we are no longer calling that anywhere! got it!